beacon_node/src/cli.rs

```diff
@@ -232,6 +232,27 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true),
         )
 
+        /*
+         * Explorer metrics
+         */
+
+         .arg(
```

Comment:
Agree on having just a single flag that accepts the monitoring endpoint.

I considered `--monitoring-endpoint` but thought that it might potentially cause some confusion with the validator monitoring flags. Agree on making it more general instead of tightly associating it with beaconcha.in.

Reply:
setting it to `--monitoring-endpoint` because not able to come up with a better name :sweat_smile:

Reply:
Nice, I'm down with `monitoring-endpoint`!